libs/utils/android/screen.py

```diff
@@ -136,4 +137,20 @@ def get_screen_density(target):
         """
         return target.execute('getprop ro.sf.lcd_density')
 
+    @staticmethod
+    def set_screen(target, on=True):
+        log = logging.getLogger('Screen')
+        if on == True:
+            target.execute('input keyevent KEYCODE_WAKE')
```

Comment:
I would really like to have this keyevent (and the following one) exposed as System APIs.
Can you add these two calls to System in a patch before this one?

```python
System.wake()
System.sleep()
```